pkgs/top-level/all-packages.nix

```diff
@@ -4780,6 +4780,8 @@ with pkgs;
 
   staruml = callPackage ../tools/misc/staruml { inherit (gnome2) GConf; libgcrypt = libgcrypt_1_5; };
 
+  otter-browser = callPackage ../applications/networking/browsers/otter {};
```

Comment:
Better use `qt5.callPackage` and specify the dependencies as arguments of your derivation.